nginx/default.conf

```diff
@@ -42,13 +42,14 @@ server {
 
   root /usr/share/nginx/html;
 
-  location / {
-    try_files $uri $uri/ /index.html =404;
-  }
-
-  location ~ ^/(.+\.(?:css|js|woff2|png))$ {
+  location ~ ^${ARLAS_WUI_APP_PATH}/(?<subpath>.+\.(?:css|js|woff2|png))$ {
     expires     30d;
     add_header  Cache-Control public;
+    try_files /$subpath =404;
+  }
+
+  location ~ ^${ARLAS_WUI_APP_PATH}/(?<subpath>.*)$ {
```

Comment:
I am worried by the mandatory `/` in the regex.

Does requests without trailing `/` work?

Example:

if:

```bash
ARLAS_WUI_APP_PATH=/abcd/efgh
```

I understand that this will work:

```
GET /abcd/efgh/
```

but I am not sure about this one:

```
```
GET /abcd/efgh
```

Reply:
Fixed